src/components/animal/detail/AnimalField.vue

```diff
@@ -3,23 +3,19 @@
     <div class="column is-4 has-background-info has-text-left">
       {{ $t('animalDetail.' + fieldData.key) }}
     </div>
-    <div class="column is-8 has-text-left" v-if="!edit" @click="edit = true">
+    <div class="column is-8 has-text-left" v-if="!edit">
       <p v-if="fieldData.value">{{ fieldData.value }}</p>
       <p v-else>{{ $t('animalDetail.noData') }}</p>
     </div>
     <div v-else class="column">
-      <div class="columns">
-        <div class="column">
-          <div class="field">
-            <div class="control">
-              <input
-                class="input is-primary"
-                type="text"
-                v-model="newValue"
-                @blur="updateField"
-              />
-            </div>
-          </div>
+      <div class="field">
+        <div class="control">
+          <input
+            class="input is-primary"
+            type="text"
+            v-model="newValue"
+            @blur="updateField"
```

Comment:
Le `@blur=updateField` risque de poser problème, si on passe d'un champ x à un champ y la valeur modifiée dans le champ x va peut-être s'annuler.

`this.newValue = this.fieldData.value;` -- Modifier/Supprimer cette ligne dans updateField (l.41)

Reply:
Pas une nao